go/builder.go

```diff
@@ -356,6 +384,9 @@ func (b *Builder) CreateByteString(s []byte) UOffsetT {
 
 // CreateByteVector writes a ubyte vector
 func (b *Builder) CreateByteVector(v []byte) UOffsetT {
+	if len(v) == 0 {
```

Comment:
Is this required?

Reply:
same reason as func (b *Builder) CreateString(s string) UOffsetT { .......